### PR TITLE
[ThreatGridv2] fix analysis_sample_command, get_dbotscore

### PR DESCRIPTION
**Pull request by michal-dagan**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: l[ink to the issue](https://jira-dc.paloaltonetworks.com/browse/XSUP-34653)

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/ThreatGrid/Integrations/ThreatGridv2/ThreatGridv2.py [hunk 1]
```diff
@@ -1035,9 +1037,9 @@ def get_dbotscore(
...
         Common.DBotScore: DBot Score according to the disposition.
     """
-    score = Common.DBotScore.NONE
-
-    if api_score >= 85:
+    if api_score == 0:
+        score = Common.DBotScore.NONE
+    elif api_score >= 85:
         score = Common.DBotScore.BAD
     elif api_score >= 50:
```
Comment (bziser) [inline Packs/ThreatGrid/Integrations/ThreatGridv2/ThreatGridv2.py:1044]: we don't want the numbers to be global values?

**Review by bziser — COMMENTED** [review]
please add a test which simulates the bug

**Review by yucohen — APPROVED** [review]

File: Packs/ThreatGrid/ReleaseNotes/2_0_21.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Cisco Secure Malware Analytics (Threat Grid) v2
+
+- Fixed an issue where XSOAR scores were considered Good when API scores were 0.
```
Comment (ShirleyDenkberg) [inline Packs/ThreatGrid/ReleaseNotes/2_0_21.md:6]:
```suggestion
- Fixed an issue where Cortex XSOAR scores were considered Good when API scores were 0.
```

**Comment by ShirleyDenkberg** [comment]
@bziser @yucohen Doc review completed.

**Pull request merged**